azurerm/resource_arm_virtual_machine_scale_set_test.go

```diff
@@ -374,6 +374,7 @@ func TestAccAzureRMVirtualMachineScaleSet_loadBalancer(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testCheckAzureRMVirtualMachineScaleSetExists("azurerm_virtual_machine_scale_set.test"),
 					testCheckAzureRMVirtualMachineScaleSetHasLoadbalancer("azurerm_virtual_machine_scale_set.test"),
+					testCheckAzureRMVirtualMachineScaleSetHasApplicationGateway("azurerm_virtual_machine_scale_set.test"),
```

Comment:
So at the moment this test will fail since there's no Application Gateway's defined in [the config used in this test](https://github.com/terraform-providers/terraform-provider-azurerm/pull/884/files#diff-62e6c44ba2b5a3617bb9c78744b7a84bR366). Could we create a separate config (as linked above) + test for creating a VMSS with an Application Gateway configured, and revert this test as-is? This'll mean we have test cases for both VMSS with an LB and an App Gateway - both of which are valid

Reply:
Makes sense. Thank you for the feedback. 
For some reason the tests were passing locally for me. Do I have to configure something locally to make the test fail?

Reply:
Oh I see. I have to run `make testacc`. Is there a way to specify an individual acceptance test to run?

Reply:
there is:

```
TF_ACC=1 go test ./azurerm -v -timeout 120m -run=TestPrefixYouWantToRun
```

where `TestPrefixYouWantToRun` is the prefix of the test you want to run (which can be a full test name) - hope that helps :)